app.py

```diff
@@ -627,6 +678,12 @@ def index(level, step):
     loaded_program_name = ''
     adventure_name = ''
 
+    if requested_lang() in ["en", "nl"]:
+        session['error_level'] = 0  # When requesting a new level, always reset error_level to 0
```

Comment:
We need to use the `set_session_var` function here as well.

Reply:
Fixed!